runtime/acala/src/lib.rs

```diff
@@ -164,6 +164,8 @@ parameter_types! {
 	pub const NftPalletId: PalletId = PalletId(*b"aca/aNFT");
 	// Vault all unrleased native token.
 	pub UnreleasedNativeVaultAccountId: AccountId = PalletId(*b"aca/urls").into_account();
+	// This Pallet is only used to payment fee pool, it's not added to whitelist by design.
```

Comment:
```suggestion
	// This Pallet is only used to payment fee pool, it's not added to whitelist by design because transaction payment pallet will ensure the accounts always have enough ED.
```